ballerina/build.gradle

```diff
@@ -220,13 +265,23 @@ publishing {
 
 updateTomlFiles.dependsOn copyStdlibs
 
+if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
```

Comment:
With this approach (line no. 268 and 281) in Windows, can we make sure both Gradle `test` and `build` tasks, start the LDAP server before tests and stop the LDAP server after tests? I just wanted to make sure that both commands (`./gradlew build` and `./gradlew test`) that a user might try does starting and stoping server as we expect. Checking this since, it is is little-bit different what we have done in other modules.

Reply:
This is there to stop running LDAP server on windows. For both `test` and `build` tasks, it doesn't run and in ubuntu for both `test`, `build` tasks, it runs. But I can change this to go inline with other modules.

Reply:
Yeah. Sorry I've typed the wrong OS. Since `test` task is executed for `./gradlew build` command also, the LDAP server will be started before tests and stopped once the `build` task is finalized. That's okay. But with `./gradlew test` command, since the `build` task is not executed, will the LDAP server get stopped? If yes, I am okay with this approach.
AFAIR, in order to cover up both scenarios we separately do start and stop the this kind of tasks related to Docker in other modules. Please refer: https://github.com/ballerina-platform/module-ballerina-auth/blob/master/ballerina/build.gradle#L164

Reply:
LDAP gets stopped with `test` also. But I will change this to be uniform with other modules.

Reply:
Okay. I just tried the same and observed that it does not get stopped.

Reply:
I tried out this in linux and it got stopped. But the logical scenario would be to not get stopped with `test`. Anyway, now it is according to the expected format.